Extension/CloudFoundry/task.loc.json

```diff
@@ -12,8 +12,8 @@
   "author": "Microsoft",
   "demands": [],
   "version": {
-    "Major": 0,
-    "Minor": 115,
+    "Major": 1,
```

Comment:
Bumped major version to prevent a clash with old UI settings and new UI workflow.